Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1

```diff
@@ -134,11 +134,11 @@ function Get-Resources
                 }
         
                 #Adding WinRMHttp port property
-                if([string]::IsNullOrEmpty($winRmHttpPortMap[$resource.Name]) -eq $false)
-                {
-                    $property = New-Object Microsoft.VisualStudio.Services.DevTestLabs.Model.PropertyBagData($false, $winRmHttpPortMap[$resource.Name])
-                    $propertyBag.Add("WinRM_Http", $property)
-                }
+                #if([string]::IsNullOrEmpty($winRmHttpPortMap[$resource.Name]) -eq $false)
+                #{
+                #    $property = New-Object Microsoft.VisualStudio.Services.DevTestLabs.Model.PropertyBagData($false, $winRmHttpPortMap[$resource.Name])
+                #    $propertyBag.Add("WinRM_Http", $property)
+                #}
 
                 #Adding WinRMHttps port property
                 if([string]::IsNullOrEmpty($winRmHttpsPortMap[$resource.Name]) -eq $false)
```

Comment:
And when we add port -can u also add Microsoft-Vslabs-MG-WinRMProtocol which would be either HTTP or HTTPS